server/tests/ballot_comparison/snapshots/snap_test_ballot_comparison_manifests.py

```diff
@@ -10,111 +10,14 @@
 snapshots[
     "test_ballot_comparison_container_manifest 1"
 ] = """Container,Tabulator,Batch Name,Ballot Number,Imprinted ID,Ticket Numbers,Already Audited,Audit Board
-CONTAINER1,TABULATOR1,BATCH1,1,1-1-1,0.243550726331576894,N,Audit Board #1
```

Comment:
Couple of questions: Why are we deleting most of this file? Also, why do manifests contain ticket numbers?

Reply:
sorry, meant to leave a comment and explain this change but then forgot! this test case was actually a tie before, so i changed it to be not a tie (see https://github.com/votingworks/arlo/pull/852/files/013ca05bcb7287382fb21c3542f5f32cf0fc9cfc#diff-0de3e56ae4db4922e32954dc0501c689b12bfeb42da260106f8e8ef323616cd0L128). so that radically changed the sample size.

i'm pretty sure this snapshot is actually of the ballot retrieval list, not the manifest

Reply:
Ah I see. Cool.